src/timedate/timedated.c

```diff
@@ -890,6 +890,10 @@ static int method_set_time(sd_bus_message *m, void *userdata, sd_bus_error *erro
         } else
                 timespec_store(&ts, (usec_t) utc);
 
+        /* when time is before systemd build date time*/
```

Comment:
Please update the comment:
- explicitly say `refuse` the request,
- missing space before the closure `*/`

So, something like
`/* refuse the request when the time is before systemd build date time */`

Reply:
done, updated